src/components/app/details/cicdHistory/Sidebar.tsx

```diff
@@ -157,6 +158,16 @@ const HistorySummaryCard = React.memo(
             return `${generatePath(path, _params)}/${currentTab}`
         }
 
+        useEffect(() => {
+            scrollToElement()
+        }, [])
```

Comment:
this Should have dependency of targetCardRef

Reply:
done